gui/slick/js/ajax-episode-subtitles.js

```diff
@@ -81,8 +81,19 @@
                         $('#releasesPP tr').eq(removeRow).remove();
                     } else {
                         // update the subtitles column with new informations
-                        language = data.subtitles;
-                        changeImage(subtitlesTd, 'images/subtitles/flags/' + language + '.png', language, language, 11, false);
+                        var language = data.subtitles;
+                        var hasLang = false;
+                        var lang = language;
+                        subtitlesTd.children().children().each(function(){
+                            // Check if user already have this subtitle language
+                            if ($(this).attr('alt').indexOf(lang) !== -1){
```

Comment:
Just replace `indexOf(lang)` with `indexOf(language)` so you can remove lang further up.